### ISSUE TITLE
#-1953801989------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1854]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-solar_lon" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-71babb76-0dcd-61ea-6d77-06bc4fe9b54d">solar_lon</div>`

<strong>Element Location: </strong>#attr-label-solar_lon

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-1953801989_All page content should be contained by landmarks.png`
